pkg/controller/pxc/controller_test.go

```diff
@@ -0,0 +1,53 @@
+package pxc
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	// "k8s.io/apimachinery/pkg/types"
+	// ctrl "sigs.k8s.io/controller-runtime"
```

Comment:
if these needs to be commented, let's delete

Reply:
done